QUDV Icon adapted

fmbouga opened the pull request:
The purpose of this issue is to somehow change the current QUDV Icon shown on creating for example a new quantity, etc. with a better Icon. 

Extra changes related to author issues has been made.

Commit:
QUDV Icon adapted

Commit:
Integrating known authors - QUDV Icon

Commit:
Revert "Integrating known authors - QUDV Icon"

This reverts commit 882c1acdf166d637d7f526f1d0de619cd34a6c88.

Commit:
Adding to known Authors

PhilMFischer commented on de.dlr.sc.virsat.qudv.ui/resources/facets-page-wizban.png:
Hi @fmbouga I think we should have an icon in the style we usually have in virtual satellite. Even though it is a nice icon i think it does not tell anything about QUDV or units in general. Maybe we should brainstorm a bit together.

PhilMFischer commented on .mailmap:
Looks like there is a merge conflict that still needs to be resolved.

PhilMFischer commented on known_authors.txt:
```diff
@@ -17,3 +17,4 @@ maur_pa
 dellerDLR
 pchrszon-dlr
 DimitriDiantos
+mbou_fr
```
your GitHub account name should be added here.

PhilMFischer reviewed (changes requested):
Please have a look to the comments. we can also work on the icon next week.

PhilMFischer commented:
Please also add a description telling us what this PR is about.

Commit:
Merge branch 'development' into feature/task_29-Update-QUDV-Dialog-Icon

fmbouga commented:
@PhilMFischer alright thanks. Yeah it is true the Icon was like trying to show some symbols, quantity and unit all together. Next week we could then either generate it together with Copilot or find another way. Thanks for the other comments related to Known authors, description

Commit:
Removed icon from Dialog - (Task #29)

After updating the icon it was not correctly rendered in the dialog.
Instead decided to remove it completely

---
Task #29: Update QUDV Dialog icons

PhilMFischer commented:
Decided to completely remove the icon from the editor, since it created issues when rendering.

Commit:
Fixing and cleaning known authors - (Task #29)

- removed old names from known authors file

---
Task #29: Update QUDV Dialog icons

Commit:
Merge branch 'development' into feature/task_29-Update-QUDV-Dialog-Icon

Commit:
Fixed mail map - (Task #29)

- fix needed due to some previously commited merge information

---
Task #29: Update QUDV Dialog icons

PhilMFischer reviewed (approved):
Looks great now- good idea to remove the icon.

PhilMFischer commented:
Closes #29
